manual/Website1/package.json

```diff
@@ -36,9 +37,9 @@
     "mini-css-extract-plugin": "^0.9.0",
     "moment": "^2.24.0",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
yarn workspaces removes the need for the `file:../` paths

Reply:
we are not using workspaces tho. Also react needs to be the same to avoid two reacts on the page (still need to add whitelisting for that)